app/background/node/service.js

```diff
@@ -16,7 +16,7 @@ import Address from 'hsd/lib/primitives/address';
 const blake2b = require('bcrypto/lib/blake2b');
 const secp256k1 = require('bcrypto/lib/secp256k1');
 const {safeEqual} = require('bcrypto/lib/safe');
-import pkg from 'hsd/lib/pkg';
+import pkg from '../../../package.json';
```

Comment:
Hm, what were we doing with `pkg` before this PR?

Reply:
It was an unsed variable.